XamarinCommunityToolkit/Views/AvatarView/AvatarView.shared.cs

```diff
@@ -1,7 +1,7 @@
-﻿using Xamarin.Forms;
-using static System.Math;
+﻿using System;
 using System.Threading;
-using System;
+using Xamarin.Forms;
+using static System.Math;
```

Comment:
in hindsight @jfversluis how do we want static namespaces to be handled (or even used)?

Reply:
I don't have a strong opinion about it, do you?

Reply:
negative